plugins/woocommerce/includes/wc-update-functions.php

```diff
@@ -416,7 +416,7 @@ function wc_update_209_brazillian_state() {
 
 	// phpcs:disable WordPress.DB.SlowDBQuery
 
-	// Update brazillian state codes.
+	// Update brazilian state codes.
```

Comment:
Shouldn't this be uppercased? (so `Brazilian`)

Reply:
Good catch, @Konamiman , thanks!
Fixed: [Uppercase the Brazilian word](https://github.com/woocommerce/woocommerce/pull/50739/commits/ac30fa22aad8dfe65dfe30310af4a889a5e50217)